setup.py

```diff
@@ -19,7 +19,7 @@
     ],
     packages=find_packages(),
     install_requires=[
-        'simso>=0.8',
+        'simso>=0.9',
```

Comment:
Why the upgrade of simso to version 0.9? simso-gui should keep working with older versions of simso

Reply:
It's not strictly required TBH. It's only useful just in case you want to tell to someone to use v0.9 if they have PyQt5.